README.md

```diff
@@ -1,143 +1,134 @@
-<div align="center">
-
-<img height="250" width="250" src="./assets/Badge_Trdelnik.png"/>
+[![Crates.io](https://img.shields.io/crates/v/trdelnik-cli?label=CLI)](https://crates.io/crates/trdelnik-cli)
+[![Crates.io](https://img.shields.io/crates/v/trdelnik-test?label=Test)](https://crates.io/crates/trdelnik-test)
+[![Crates.io](https://img.shields.io/crates/v/trdelnik-client?label=Client)](https://crates.io/crates/trdelnik-client)
+[![Crates.io](https://img.shields.io/crates/v/trdelnik-explorer?label=Explorer)](https://crates.io/crates/trdelnik-explorer)
+<br />
+[![lint](https://github.com/Ackee-Blockchain/trdelnik/actions/workflows/lint.yml/badge.svg)](https://github.com/Ackee-Blockchain/trdelnik/actions/workflows/lint.yml)
+[![test](https://github.com/Ackee-Blockchain/trdelnik/actions/workflows/test.yml/badge.svg)](https://github.com/Ackee-Blockchain/trdelnik/actions/workflows/test.yml)
```

Comment:
It looks like it's not positioned properly
<img width="782" alt="Screenshot 2022-06-06 at 22 45 47" src="https://user-images.githubusercontent.com/41209053/172246361-e8b62075-b7d4-47ce-a034-57153a2ed8be.png">

Reply:
I moved it under the logo